corehq/pillows/xform.py

```diff
@@ -214,8 +212,9 @@ def get_xform_pillow(pillow_id='xform-pillow', ucr_division=None,
     )
     unknown_user_form_processor = UnknownUsersProcessor()
     form_meta_processor = FormSubmissionMetadataTrackerProcessor()
-    checkpoint_id = "{}-{}-{}-{}".format(
-        pillow_id, XFORM_INDEX_INFO.index, REPORT_XFORM_INDEX_INFO.index, USER_INDEX)
+    # TODO Do we need to preserve this checkpoint format?
```

Comment:
We can copy the existing checkpoints to the new key (maybe with a migration?), to make sure the new pillow picks up where the old one left off.